include/internal/catch_common.hpp

```diff
@@ -22,7 +24,7 @@ namespace Catch {
         return s.find( infix ) != std::string::npos;
     }
     void toLowerInPlace( std::string& s ) {
-        std::transform( s.begin(), s.end(), s.begin(), ::tolower );
+        std::transform( s.begin(), s.end(), s.begin(), static_cast<int(*)(int)>(std::tolower) );
```

Comment:
std::tolower should already be `int std::tolower(int)` - so the cast is a best superfluous, and at worst is undefined behaviour at worst.